optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/OptaPlannerProcessor.java

```diff
@@ -217,6 +217,7 @@ private void generateConstraintVerifier(SolverConfig solverConfig,
             final Class<?> constraintProviderClass = solverConfig.getScoreDirectorFactoryConfig().getConstraintProviderClass();
             final Class<?> planningSolutionClass = solverConfig.getSolutionClass();
             final List<Class<?>> planningEntityClasses = solverConfig.getEntityClassList();
+            // TODO Don't duplicate defaults by using ConstraintVerifier.create(solverConfig) instead
```

Comment:
I tried to also use ConstraintVerifier.create(solverConfig) to reduce the amount of code here significantly (including no longer duplicating the default settings (DROOLS etc), but I didn't see how to transfer the solverConfig instance into the generated code.

@Christopher-Chianelli Might you have an idea?

Reply:
Maybe create a `OptaplannerTestBeanProvider` (needed as optaplanner-test is optional) which construct the constraint verifier (and would have access to the injected `SolverConfig` instance)?